pages/initiatives/gsoc/gsoc2021ideas.md

```diff
@@ -206,4 +206,36 @@ The OWASP SecureTea Project provides a one-stop security solution for various de
 * [Rejah Rehim](mailto:rejah.rehim@owasp.org)
 * [Ade Yoseman](mailto:edikdoank@gmail.com)
 
+<hr>
+
+### [OWASP Intelligent Intrusion Detection System](https://github.com/OWASP/Intelligent-Intrusion-Detection-System)
+
+##### Explanation of Ideas
+
+OWASP IIDS is an open source software that leverages the benefits of Artificial Intelligence to detect the intrusion and alert the respective network administrator.
+
+##### Getting started
+
+* You may read the available documents in the [wiki page](https://owasp.org/www-project-intelligent-intrusion-detection-system/).
```

Comment:
Someone should probably actually fix this content if you're going to refer to it.
- Broken images
- Raw/Broken HTML
- "Project Resources" duplicate links that go nowhere